### PR TITLE
UserVoiceShow: Fixed for simplified profile

### PR DESCRIPTION
**Pull request by LordEliasTM**
After multiple requests to fix my plugin i actually did. Please merge, i'm tired.

Resolves Issue #2664 
Resolves Pulls #2798 #2665 #1783 #1669 
#### Fixed:
- [x] User profile popout patch
- [x] User profile modal patch
#### Removed:
- [x] Setting to disable header
#### Added:
- [x] Button to view the voice channel (eye)
- [x] Decorator to view channel when clicked (blue speaker)

## Images
image
image
image

### CODE REVIEW

File: src/plugins/userVoiceShow/index.tsx [hunk 1]
```diff
@@ -73,28 +88,69 @@ export default definePlugin({
     authors: [Devs.LordElias],
     settings,
 
+    start() {
+        VoiceStateStore.getAllVoiceStates(); // reduces api spam i hope
```
Comment (Nuckyz) [inline src/plugins/userVoiceShow/index.tsx:92]:
```suggestion
        VoiceStateStore.getAllVoiceStates(); // reduces api spam i hope
```

This returns an object and makes no API calls

**Review by Nuckyz — CHANGES_REQUESTED** [review]

**Comment by Nuckyz** [comment]
I have some ideas on how to make this fit simplified profiles better. Will work on it and show the results

File: src/plugins/userVoiceShow/index.tsx (hunk 1 above)
Comment (LordEliasTM) [reply src/plugins/userVoiceShow/index.tsx:92]: So i can remove that line and be fine?

Comment (Nuckyz) [reply src/plugins/userVoiceShow/index.tsx:92]: yes

**Comment by Nuckyz** [comment]
Superseded by https://github.com/Vendicated/Vencord/compare/67dbbec16880...bae9f8cbe696, thanks a lot anyways!

**Comment by LordEliasTM** [comment]
Yea f u too

**Pull request closed without merging**